scripts/build-isa-sim.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
```

Comment:
what is this script doing? Installing spike?

Reply:
yes. in my experience, i have rebuilt the simulator 10 times for every toolchain build when working on a RoCC accelerator